docs/DeveloperGuide.adoc

```diff
@@ -1869,3 +1871,14 @@ To achieve this, you can filter deliveryman by name, then assign him a `Pending`
 ... Test case: `/deliveryman select 0` +
     Expected: No deliveryman would be selected and displayed as this is an invalid index.
 
+=== Finding deliveryman
+.. Prerequisite: Logged into the application. Having at least one deliveryman.
+
+... Test case: `/deliveryman find n/tom` +
+    Expected: Deliveryman with the name `tom`, `tommy`, `tom hanks` would be displayed on the display panel.
+
+... Test case: `/deliveryman find n/tom n/john` +
+    Expected: Deliveryman with the name `john`, `johnny`, `john doe` would be displayed on the display panel.
```

Comment:
Probably should mention that the parser takes in the last data instead of all the data.